dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxws.3.2/src/org/apache/cxf/jaxws/JaxWsClientProxy.java

```diff
@@ -213,17 +238,28 @@ boolean isAsync(Method m) {
     }
 
     static SOAPFault createSoapFault(SOAPBinding binding, Exception ex) throws SOAPException {
+        
+        boolean isFinestEnabled = LOG.isLoggable(Level.FINEST);   // Liberty Change issue #26529
         SOAPFault soapFault;
         try {
             soapFault = binding.getSOAPFactory().createFault();
+            if(isFinestEnabled)  {
+                LOG.finest("SOAPFault is created trough SOAPBinding SOAPFactory: " + soapFault);   // Liberty Change issue #26529
```

Comment:
Will soapFault log the fault message or do we need to call soapFault.getMessage()?